daycare/test_image.py

```diff
@@ -20,3 +22,13 @@ def test_image_creation_with_invalid_image_url(self, create_daycare_user):
         with pytest.raises(ValidationError,
                            match="Invalid URL image - URL should end with \'.gif\', \'.png\', \'.jpg\' or \'.jpeg\'."):
             Image.create(url="NOT_VALID_URL", daycare_id=DayCare.objects.get(id=create_daycare_user.id))
+
+    def test_daycare_has_customized_profile_image(self, create_image1, create_image2, create_daycare_user):
+        daycare_profile_image = create_daycare_user.get_daycare_primary_image_url()
+        assert daycare_profile_image != DEFAULT_DAYCARE_PROFILE_URL
+        assert daycare_profile_image is not None
+
+    def test_daycare_has_default_profile_image(self, create_daycare_user):
```

Comment:
would expect a better name. *why* does it have the default profile_image? (test_......_when_______)

Reply:
great!